production/technical_userguide.md

```diff
@@ -530,3 +531,14 @@ For ease of use, we have listed the `cURL` commands for the operation below. The
 curl https://api.bcda.cms.gov/data/48/4e2cd98c-4746-4138-872b-24778c000b02.ndjson
 -H 'Authorization: Bearer {token}'
 ```
+
+## Generate Client Code
+
+There is an OpenAPI json file available which can be used to generate client code to make integrating with BCDA easier. Take the JSON file from [here](https://sandbox.bcda.cms.gov/api/v1/swagger/openapi.json) and upload it to [https://editor.swagger.io/](https://editor.swagger.io/) to create a client with the offical Swagger tool. Then select the preferred language from the `Generate Client` dropdown to create and download an API client with README. This tool can be used to generate clients in Java, Python, Go, and other languages.
```

Comment:
References to `sandbox` here but shouldn't we instead be pointing them to `https://api.bcda.cms.gov` since this is the production user guide?